app/presenters/service_standard_presenter.rb

```diff
@@ -30,6 +30,7 @@ def links_payload
     {
       links: {
         primary_publishing_organisation: [ServiceManualPublisher::GDS_ORGANISATION_CONTENT_ID],
+        email_alert_signup: [],
```

Comment:
Is it worth having this in a separate PR to revert? Doesn't feel like it supports republishing really.

Reply:
Sure, I've split that out: https://github.com/alphagov/service-manual-publisher/pull/815

Reply:
👍